src/Button/ToggleGroup/ToggleGroup.tsx

```diff
@@ -30,6 +30,8 @@ interface BaseProps {
   /**
    * The value fo the `name` attribute of the children to select/press
    * initially.
+   * Note: This prop will have full control over the pressed prop on its children. Setting select/pressed on the
+   * children props directly, will have no effect.
```

Comment:
```suggestion
   * children props directly will have no effect.
```

Reply:
I am not sure here :) but it should work without `,`

Reply:
It's correct. Better without comma